src/Core/Vipr.Core/CodeModel/OdcmCapabilities.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vipr.Core.CodeModel
+{
+    public class OdcmCapabilities
+    {
+        public bool Insertable { get; set; }
```

Comment:
>" There's not going to be custom parsing semantics for annotations that are already defined in some schema somewhere."

This is not true. For example take the case UpdateRestriction annotation. We do not directly parse and convert this annotation into strongly typed CLR object using the generic parser. That is not very useful in how we want to represent OdcmCapability for OdcmProperty. Instead we do custom parsing to represent this annotation in the OdcmModel.